core/jazz_environment-event-handler/config/test-config.json

```diff
@@ -1,26 +1,56 @@
 {
-    "BASE_API_URL": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com",
-    "ENVIRONMENT_API_RESOURCE": "/dev/jazz/environments",
-    "DELETE_ENVIRONMENT_API_RESOURCE": "/dev/jazz/delete-serverless-service",
-    "TOKEN_URL": "/dev/jazz/login",
-    "SERVICE_USER": "{jazz_admin}",
-    "TOKEN_CREDS": "{jazz_admin_creds}",
-    "EVENTS": {
-        "EVENT_TYPE": ["SERVICE_DEPLOYMENT", "SERVICE_ONBOARDING", "SERVICE_DELETION"],
-        "EVENT_NAME": ["COMMIT_TEMPLATE", "CREATE_BRANCH", "UPDATE_ENVIRONMENT", "DELETE_BRANCH", "DELETE_ENVIRONMENT"],
-        "UPDATE_ENVIRONMENT": "UPDATE_ENVIRONMENT",
-        "INITIAL_COMMIT": "COMMIT_TEMPLATE",
-        "CREATE_BRANCH": "CREATE_BRANCH",
-        "DELETE_BRANCH": "DELETE_BRANCH",
-	"DELETE_ENVIRONMENT": "DELETE_ENVIRONMENT"
-    },
-    "ENVIRONMENT_CREATE_REQUIRED_FIELDS": ["domain", "branch", "created_by", "service_name"],
-    "ENVIRONMENT_UPDATE_REQUIRED_FIELDS": ["domain", "service_name"],
-    "ENVIRONMENT_PRODUCTION_PHYSICAL_ID": "master",
-    "CREATE_ENVIRONMENT_STATUS": "inactive",
-    "ENVIRONMENT_DELETE_STARTED_STATUS": "deletion_started",
-    "ENVIRONMENT_DELETE_FAILED_STATUS": "deletion_failed",
-    "ENVIRONMENT_DELETE_COMPLETED_STATUS": "archived",
-    "RANDOM_CHARACTERS": "0123456789abcdefghijklmnopqrstuvwxyz",
-    "RANDOM_ID_CHARACTER_COUNT": 10
+  "BASE_API_URL": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com",
+  "ENVIRONMENT_API_RESOURCE": "/dev/jazz/environments",
+  "DELETE_ENVIRONMENT_API_RESOURCE": "/dev/jazz/delete-serverless-service",
+  "TOKEN_URL": "/dev/jazz/login",
+  "SERVICE_API_RESOURCE": "/prod/jazz/services",
```

Comment:
this should be non-prod one, perhaps: `/dev/jazz/services`

Reply:
done